.gitignore

```diff
@@ -2,3 +2,12 @@ elm-stuff/
 node_modules/
 tmp/
 dist/
+
+#IDE configuration files
```

Comment:
I'm not a huge fan of IDE and OS specific stuff in project's `.gitignore` files. Imo this sort of things should be handled [in a global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) per machine. But after all that's just personal taste, so definitely up for discussion.